WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormActionsFactory.swift

```diff
@@ -95,6 +99,11 @@ struct ProductFormActionsFactory: ProductFormActionsFactoryProtocol {
         return visibleSettingsSectionActions()
     }
 
+    /// Returns an array of actions that are visible in the product form options CTA section.
+    func optionsCTASectionActions() -> [ProductFormEditAction] {
+        isAddOptionsButtonEnabled ? [.addOptions] : []
+    }
```

Comment:
We need to add some restrictions to this, because the "Add options" row is meant as a prompt to add product variations. It's only meant to appear when the "Add product variations" row is also visible in the "add more details" bottom sheet, and should otherwise be hidden. I've opened #8927 to follow up on that.